apps/wallet/src/ui/app/pages/home/receipt/index.tsx

```diff
@@ -33,13 +31,10 @@ function ReceiptPage() {
         { enabled: !!transactionId, retry: 8 }
     );
 
-    // return route or default to transactions
-    const linkTo = fromRoute ? `/${fromRoute}` : '/transactions';
```

Comment:
do we use this anywhere?

Reply:
This was removed

Reply:
I mean - is there a need for this anymore? I don't see anything routing with a `from=` param, but just double checking that isn't needed